### DISCUSSION TITLE
Postgres JsonB datastructure in dev/test but string in prod

### DISCUSSION THREAD

**Discussion by currentoor** [prisma/prisma#18011] [Q&A]
I have this schema
```
model task {
  id                       String        @id @default(uuid()) @db.Uuid
  custom_fields            Json?         @db.JsonB
}
```

When I query using
```js
await prisma.task.findUnique({...})
```

In dev/test `custom_fields` is a datastructure, but in prod it's a string.

Is this a config change I need to make somewhere? Ideally it would always be parsed into a datastructure.

**Comment by nurul3101** [comment]
Hello @currentoor 👋 

Does your dev and prod environments both have the same configuration?
Like same Prisma, node and database versions?

Can you try logging the value of custom_fields with [typeof](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof) in both environments?

**Reply by currentoor** [reply]
Prisma and node are the same versions but I did notice in prod postgres is 13.7 whereas in dev it's 14.6, do you think that could be the issue?

```
typeof custom_fields => string // prod
typeof custom_fields => object // dev
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
